moped-editor/src/auth/user.js

```diff
@@ -26,13 +27,6 @@ export const UserContext = createContext(null);
  */
 export const atdColorKeyName = "atd_moped_user_color";
 
-/**
- * This is a constant string key that holds the profile for a user.
- * @type {string}
- * @constant
- */
-export const atdSessionKeyName = "atd_moped_user_context";
```

Comment:
This PR also removes the a local storage entry called `atd_moped_user_context` which was another copy of the user session that can be retrieved using the Amplify library. 

It took some figuring out, but this value was keeping users at their current route (like `/projects/1392?tab=notes`) when the browser refreshed since it was the initial value of the `user` state. The app would not redirect to the sign in screen as long as the `user` state was populated. 

The initial value of the `user` state is now set using Amplify, and the redirect to previous route is handled by React Router state that I'll comment on later in this PR.